FirebaseAppCheck/Tests/Unit/Core/FIRAppCheckTests.m

```diff
@@ -556,7 +556,29 @@ - (void)testTokenRefreshTriggeredAndRefreshError {
   [self verifyAllMocks];
 }
 
-#pragma mark - Token update notifications
+- (void)testLimitedUseToken {
```

Comment:
Should there also be unit tests for the following cases:
```objc
- (void)testLimitedUseToken_WhenTokenGenerationErrors { /* ... */ }
- (void)testLimitedUseTokenDoesNotUseCachedToken { /* ... */ }
```

Reply:
I think the limited-use token is not cached defaultly, so we don't need to add - (void)testLimitedUseTokenDoesNotUseCachedToken { /* ... */ }?  And for - (void)testLimitedUseToken_WhenTokenGenerationErrors { /* ... */ } sure, adding now. Sry for sending it for review before add them all together. pretty new to ios and to appCheck, so need some check first. And thanks for the detailed comments!

Reply:
Thanks for pointing that out. I have modified the test to not use cashed tokens and add errorTest too.